code/modules/mob/living/simple_animal/constructs.dm

```diff
@@ -46,7 +46,13 @@
 	var/can_repair_constructs = FALSE
 	var/can_repair_self = FALSE
 	var/runetype
-	var/theme = THEME_CULT //cult is classic red, holy is blue, and wizard is purple
+/*
```

Comment:
This isn't in the doc comment format.

https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md